src/body.rs

```diff
@@ -22,19 +22,15 @@ impl Body {
     /// A `Body` constructed from a set of bytes, like `String` or `Vec<u8>`,
     /// are stored differently and can be reused.
     pub fn new<R: Read + Send + 'static>(reader: R) -> Body {
-        Body {
-            reader: Kind::Reader(Box::new(reader), None),
```

Comment:
I find it preferable to be on multiple lines in these cases, as it makes future diffs less disruptive (if a new property is ever added to `Body`).